GlobalStaticVersion.props

```diff
@@ -6,14 +6,14 @@
       Update for every public release. 
     -->
     <SemanticVersionMajor>2</SemanticVersionMajor>
-    <SemanticVersionMinor>1</SemanticVersionMinor>
-    <SemanticVersionPatch>1</SemanticVersionPatch>
+    <SemanticVersionMinor>2</SemanticVersionMinor>
+    <SemanticVersionPatch>0</SemanticVersionPatch>
     <!--PreReleaseMilestone>beta1</PreReleaseMilestone-->
     <!-- 
       Date when Semantic Version was changed. 
       Update for every public release.
     -->
-    <SemanticVersionDate>2016-06-28</SemanticVersionDate>
+    <SemanticVersionDate>2016-12-07</SemanticVersionDate>
```

Comment:
I think that this time the date change needs to happen because there have been no prior 2.2.0-beta* builds and we did change the minor version just now.